applications/nrf_desktop/sample.yaml

```diff
@@ -23,40 +24,56 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54l15pdk@0.3.0/nrf54l15/cpuapp
-    tags: bluetooth ci_build
+    tags: bluetooth ci_build sysbuild
   applications.nrf_desktop.zdebug_wwcb:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_wwcb.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=wwcb
   applications.nrf_desktop.zdebug_fast_pair.gmouse:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840 nrf52840gmouse/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52840gmouse/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_fast_pair.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=fast_pair
```

Comment:
for `nrf52840dk/nrf52840` target:
`SB_CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=n` might be missing compared to the `prj_fast_pair.conf`

Reply:
There is also an issue with disabling `SB_CONFIG_SECURE_BOOT_APPCORE` on `nrf52840dk/nrf52840`:
```
warning: SECURE_BOOT_APPCORE (defined at
/home/mak4/ncs/nrf/applications/nrf_desktop/Kconfig.sysbuild:22,
/home/mak4/ncs/nrf/sysbuild/Kconfig.secureboot:13) was assigned the value 'n' but got the value 'y'.
See http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SECURE_BOOT_APPCORE and/or look up
SECURE_BOOT_APPCORE in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.
```
Seems to be present on all `nrf52840dk` configs which enables mcuboot.